lib/validator/QubitValidatorCountable.class.php

```diff
@@ -36,6 +36,6 @@ protected function doClean($value)
      */
     protected function isEmpty($value)
     {
-        return 1 > count($value);
+        return 1 > count((array) $value);
```

Comment:
I think 

```PHP
return empty($value);
```

is equivalent and easier to read.  

I wonder why `sfValidatorBase` has an `isEmpty()` method at all instead of just using the built-in `empty()` function?  Maybe `empty()` has slightly different handling of "falsey" values?